apps/marketplace/components/BuyerSpecialist/BuyerSpecialistSelectStage.js

```diff
@@ -13,10 +13,10 @@ import { countLabourHireSellers } from 'marketplace/actions/supplierActions'
 
 import mainStyles from '../../main.scss'
 
-const requiredCategory = v => v.sellerCategory
-const requiredChoice = v => !v.sellerCategory || v.openTo
+const requiredChoice = v => v.openTo
 const atLeastOneSeller = v =>
   !v.openTo || v.openTo === 'all' || (v.openTo === 'selected' && v.sellers && Object.keys(v.sellers).length > 0)
+const requiredCategory = v => v.sellerCategory
```

Comment:
It looks like this line hasn't changed other than the position in the file.  I'd try and avoid making changes like this in the future as it can make it a bit harder to review incoming changes.

Reply:
Noted- it does make it more cumbersome comparing changes.